package problema_3.model.exception;

public class CPFInvalidoException {

}
